style.css

```diff
@@ -0,0 +1,349 @@
+*{
+    margin: 0;
+    padding: 0;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+:root{
+    --primary-color:#3B82F6;
+    --secondary-color:#6B7280;
+    --white-color:#E5E7EB;
+    --grey: rgb(206, 206, 206)
+}
+/* colors */
+.red-bg{
+    background: red;
+}
+.yellow-bg{
+    background: yellow;
+}
+.blue-bg{
+    background: blueviolet;
+}
+.red-cl{
+    color: red;
+}
+.yellow-cl{
+    color: yellow;
+}
+.blue-cl{
+    color: blueviolet;
+}
+.red-light{
+    background: rgb(255, 155, 155);
+}
+.yellow-light{
+    background: rgb(255, 255, 173);
+}
+.blue-light{
+    background: rgb(156, 156, 236);
+}
+
+.pd-1{
+    padding: 1rem;
+}
+.pd-2{
+    padding: 2rem;
+}
+
+.sm {
+    height: 3rem;
+}
+.md{
+    height: 5rem;
+}
+.lg{
+    height: 7rem;
+}
+
+/* Avatar */
+.avatar{
+    border-radius: 50%;
+}
+
+/* Avator badge */
+.avatar-badge{
+    position: relative;
+    display: flex;
+    flex-direction: column-reverse;
+}
+.badge{
+    border-radius: 50%;
+    border: 0.2rem solid white;
+    position: absolute;
+}
+.badge-sm{
+    height: 0.5rem;
+    width: 0.5rem;
+    border: 0.1rem solid white;
+    bottom: 0.5rem;
+    left: 2.5rem;
+}
+.badge-md{
+    height: 0.8rem;
+    width: 0.8rem;
+    border: 0.2rem solid white;
+    bottom: 1rem;
+    left: 4.5rem;
+}
+.badge-lg{
+    height: 1rem;
+    width: 1rem;
+    border: 0.2rem solid white;
+    bottom: 1rem;
+    left: 6rem;
+}
+/* Icon badge */
+.icon-div{
+    position: relative;
+    
+}
+.icon-count{
+border-radius: 50%;
+}
+.icon-count-sm{
+    font-weight: 600;
+    font-size: 1.2rem;
+    padding: 0.1rem 0.3rem;
+    position: absolute;
+}
+.icon-count-md{
+    font-weight: 700;
+    font-size: 1.3rem;
+    padding: 0.1rem 0.3rem;
+    position: absolute;
+}
+.icon-count-lg{
+    font-weight: 700;
+    font-size: 1.4rem;
+    padding: 0.1rem 0.3rem;
+    position: absolute;
+}
+
+/* Alerts */
+.alert{
+    padding: 1rem 3rem;
+    display: inline-block;
+    border-radius: 5px;
+}
+
+/* button */
+.btn{
+    padding: 0.8rem 1.2rem;
+    border-radius: 3px;
+    color: white;
+    font-weight: 600;
+    font-size: 1.3rem;
+    cursor: pointer;
+    opacity: 0.8;
+}
+.btn-primary{
+    background-color: var(--primary-color);
+}
+.btn-secondary{
+    background-color: var(--secondary-color);
+}
+.btn-link1{
+    color: var(--primary-color);
+}
+.btn-link2{
+    color: var(--secondary-color)
+}
+.btn-link1:hover{
+    text-decoration: underline;
+}
+.btn-link2:hover{
+    text-decoration: underline;
+}
+.btn:hover{
+    opacity: 1;
+}
+.cta{
+    background: var(--primary-color);
+    font-size: 1.5rem;
+    border-radius: 50%;
+}
+
+/* Card vertical */
+.card{
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    max-width: 15rem;
```

Comment:
fixed max-width might create an issue in responsiveness

Reply:
Done